content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -111,6 +112,14 @@
       &.products {
         background-image: url('#{$image-path}trailhead/benefit-products.png');
       }
+
+      &.sync {
+        background-image: url('#{$image-path}trailhead/benefit-sync.png');
+      }
+
+      &.lockwise {
+        background-image: url('#{$image-path}trailhead/benefit-lockwise.png');
```

Comment:
This image seems to be the same as `benefit-privacy.png` so probably just reuse that one instead of adding the style and file. Unless the lockwise image should be something different?

Reply:
good catch